src/transformers/cache_utils.py

```diff
@@ -488,6 +492,13 @@ class QuantoQuantizedCache(QuantizedCache):
 
     def __init__(self, cache_config: CacheConfig) -> None:
         super().__init__(cache_config)
+        quanto_version = version.parse(importlib.metadata.version("quanto"))
+        if quanto_version < version.parse("0.2.0"):
+            raise ImportError(
+                f"You need quanto package version to be greater or equal than 0.2.0. Detected version {quanto_version}. "
```

Comment:
nit - just because users sometimes look just at the warning and not the traceback - this should save use some issues :) 

```suggestion
                f"You need quanto package version to be greater or equal than 0.2.0 to use `QuantoQuantizedCache`. Detected version {quanto_version}. "
```